programs/server/Server.cpp

```diff
@@ -213,6 +214,10 @@ void setupTmpPath(Poco::Logger * log, const std::string & path)
             LOG_DEBUG(log, "Skipped file in temporary path {}", it->path().string());
     }
 }
+catch (...)
+{
+    DB::tryLogCurrentException(log, "Caught exception while setup temporary path: " + path);
```

Comment:
Add a comment:

"It is ok to skip this exception as cleaning old temporary files is not necessary."

Otherwise catching exception looks suspicious.